cpp/ql/lib/semmle/code/cpp/models/interfaces/Throwing.qll

```diff
@@ -11,12 +11,21 @@ import semmle.code.cpp.models.Models
 import semmle.code.cpp.models.interfaces.FunctionInputsAndOutputs
 
 /**
- * A class that models the exceptional behavior of a function.
+ * A function that is known to raise an exception.
  */
 abstract class ThrowingFunction extends Function {
+  ThrowingFunction() { any() }
+
```

Comment:
This doesn't seem necessary?

Reply:
I once had a nuanced reason for this on the previous iteration, but with our recent changes this is vestigial. Fixed